.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+<!--
+We appreciate your contribution to the Jaeger project! 👋🎉
+
+Before creating a pull request, please make sure:
+- Your PR is solving one problem
+- You have read the CONTRIBUTING.md guide
+- You signed all your commits (otherwise we won't be able to merge the PR)
```

Comment:
Can you add hyperlink here to https://github.com/jaegertracing/jaeger/blob/master/CONTRIBUTING.md#sign-your-work

 I know the above point will cover this but less friction the better?

Reply:
added, but it looks a lot busier with links. Because it's a comment I can't actually hide the links.